webui/src/lib/components/navbar.jsx

```diff
@@ -20,7 +20,7 @@ const NavUserInfo = () => {
         <Navbar.Collapse className="justify-content-end">
             <NavDropdown title={user.friendly_name || user.id} className="navbar-username" alignRight>
                 <NavDropdown.Header>
-                    User: <code>{user.accessKeyId}</code>
+                    User: <code>{user.accessKeyId || user.id}</code>
```

Comment:
So this will sometimes show the user ID and sometime the access key? Better just not show anything.

Reply:
@johnnyaug By not showing anything you mean Not showing anything when there is no accessKey or Not showing anything at all?

Reply:
I say remove it altogether, otherwise strange behaviors can be expected.